dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/KeyValuePair.java

```diff
@@ -24,12 +24,20 @@
 
     public boolean isString();
 
+    public boolean isFloat();
+
+    public boolean isBoolean();
+
     public Integer getIntValue();
 
     public Long getLongValue();
 
     public String getStringValue();
 
+    public Float getFloatValue();
```

Comment:
these getX methods should all return native types (int, long, String, float, bool) -- this change will require corresponding change to all implementation classes too.